src/unstract/adapters/vectordb/pinecone/src/pinecone.py

```diff
@@ -1,11 +1,10 @@
 import logging
 import os
-import time
 from typing import Any, Optional
 
 import pinecone
```

Comment:
@gaya3-zipstack from which dependency does this import come from? I saw a few `pinecone-client` libs removed in the `pyproject.toml`. Its best to directly specify the dependencies that we use even if its resolved transitively through a library

Reply:
There was a sleep here in the code. I removed the sleep and hence the import of time. With llama-index-0.10.x, the explicit llama-index-<adapter> packages should take care of the dependencies. We will come to know more with developer testing. I have not tested all adapter instance creation with this.